tasks/CleanUp.tex

```diff
@@ -1,67 +1,71 @@
 \section{Clean Up [Housekeeper]}
-The robot has to clean the room from the house it has been told to clean up.
+The robot has to tidy up a room inside the arena.
 
 % \subsection{Focus}
 % This test focuses on object perception, manipulation, and planning.
 
 \subsection{Main Goal}
-Upon entrance, the robot requests the operator which room shall be cleaned.
-All misplaced known objects found in this room must be taken to their predefined locations and unknown objects thrown to the trash bin.
+Inside one room in the arena are some misplaced objects. The robot has to find each one and bring them to their predefined locations. Unknown objects have to be thrown into the bin. Near the arena entrance will be an operator who can tell the robot which room is messy and will open the entrance door if the robot requests it.
```

Comment:
- There are some misplaced objects in one of the arena's rooms.
- Unknown objects are thrown to the bin

@justinhart  maybe you can give later a buff